stdpopsim/ext/selection.py

```diff
@@ -68,6 +68,15 @@ def __attrs_post_init__(self):
                     "Weibull-distributed sel. coef. (distribution_type='w') "
                     "use a (scale, shape) parameterisation, requiring parameters > 0."
                 )
+        elif self.distribution_type == "l":
+            # An lognormally-distributed fitness effect (mean, standard deviation).
+            # See Eidos documentation for rlnorm().
+            if len(self.distribution_args) != 2 or self.distribution_args[1] <= 0:
+                raise ValueError(
+                    "Lognormally-distributed sel. coefs. (distribution_type='l') "
+                    "use a (logmean, logsd) parameterisation, requiring logsd > 0."
+                )
+            self.distribution_type = "s"
```

Comment:
How about we set `self.distribution_args = "{return Q * rlnorm(...)}"` here, instead of in the other file? That should make it easier if in the future we decide to add some other distribution type that also uses the "s" type with a source string.